Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Base image
+FROM ubuntu:20.04
```

Comment:
I think we should use the Nvidia container toolkit so we can leverage NVIDIA GPUs

Reply:
sure i will take care of that